apps/arweave/src/ar_mining_worker.erl

```diff
@@ -13,24 +13,24 @@
 -include_lib("eunit/include/eunit.hrl").
 
 -record(state, {
-	name						= not_set,
-	partition_number			= not_set,
-	diff_pair					= not_set,
-	packing_difficulty			= 0,
-	task_queue					= gb_sets:new(),
-	active_sessions				= sets:new(),
-	%% The sub_chunk_cache stores either the first or second sub-chunk for a given nonce. This
-	%% is because we process both the first and second recall ranges in parallel and don't know
-	%% which data will be available first. For spora_2_6 packing (aka difficulty 0), sub-chunks
-	%% and chunks are the same size (256KiB), for composite packing each sub-chunk is 8KiB.
-	sub_chunk_cache 			= #{},
-	sub_chunk_cache_size		= #{},
-	sub_chunk_cache_limit		= 0,
-	vdf_queue_limit				= 0,
-	latest_vdf_step_number		= 0,
-	is_pool_client				= false,
-	h1_hashes					= #{},
-	h2_hashes					= #{}
+    name = not_set :: atom(),
+    partition_number = not_set :: non_neg_integer() | not_set,
+    diff_pair = not_set, % :: ?DIFF_PAIR()
```

Comment:
@vird what is the diff pair here?

Reply:
A pair of mining difficulties, the first element is the mining difficulty for one-chunk solutions, the second element - for two-chunk solutions.